doc/functions/overrides.xml

```diff
@@ -6,8 +6,16 @@
 
  <para>
   Sometimes one wants to override parts of <literal>nixpkgs</literal>, e.g.
-  derivation attributes, the results of derivations or even the whole package
-  set.
+  derivation attributes, the results of derivations.
+ </para>
+
+ <para>
+  These overriding functions let you focus on one part of Nixpkgs and give you
+  back the requested variation. This is orthogonal but related to overlays and
+  the extending functions. Those also let you make modifications but return the
+  whole package set instead of just what you modified. When used together, the
```

Comment:
This paragraph is not cristal clear to me when I read it. I think you meant something along the lines of: "These functions are used to make changes to a single package, while overlays can combine these changes across the entire set of packages of Nixpkgs"

Also, consider adding a link to the overlay section within the documentation.

Reply:
Thanks, I have improved the writing and added a link.